README

```diff
@@ -1130,10 +1130,10 @@ NETWORKING SUPPORT / OPTIONS
   details.
 
 --with-libfabric=<directory>
-  Specify the directory where the OpenFabrics Interfaces libfabric
-  library and header files are located.  This option is generally only
-  necessary if the libfabric headers and libraries are not in default
-  compiler/linker search paths.
+  Specify the directory where the libfabric library and header files
+  are located if not present in default compiler/linker search paths.
```

Comment:
Why did you change this paragraph?

* Is there a reason to remove "OpenFabrics Interfaces"?
* Is there a reason to remove "This option is generally ..."?  I think several other options echo similar language.

Reply:
It is a bit too verbose IMO. The `... OpenFabrics Interfaces libfabric...` wording is actually redundant given the next paragraph explains `Libfabric is the support library for OpenFabrics Interfaces-based network adapters...` so I think just calling it libfabric should be sufficient.

I could be wrong, but `This option is generally only necessary...` language is ambiguous. Really the only scenario where the user needs `--with-libfabric` is when libfabric is not installed in any of the canonical locations.

Reply:
Can you please leave the rest of the text alone?  We're not looking to heavily revise the 4.1 README at this point.  We're just looking for the language about older Open MPI versions and libfabric v2.0.x.